lib/crawly/utils.ex

```diff
@@ -224,4 +224,27 @@ defmodule Crawly.Utils do
   def ensure_loaded?(module) do
     Code.ensure_loaded?(module)
   end
+
+  def unwrap_module_and_options(
```

Comment:
didn't spot this before. Could you adjust this function so that it only unwraps a provided arg? So that we can just pipe the result of `Utils.get_settings` into the function: 
```elixir
Utils.get_settings(:fetcher)
|> Utils.unwrap_module_and_options()
```
also can you add a docs and example usage

Reply:
and also add typespecs. Function should take a term as first arg.

Reply:
as to why this would be better, it reduces the interface of the function and aligns the function name with what it actually does.

Reply:
done